src/pure.test.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const SCRIPT_SELECTOR =
+  'script[src="https://js.stripe.com/v3"], script[src="https://js.stripe.com/v3/"]';
+
+describe('pure module', () => {
+  afterEach(() => {
```

Comment:
Should we not have a more thorough cleanup like in `index.test.ts` ?
In particular:
```js
    delete window.Stripe;
    jest.resetModules();
```

Right now we rely on the all the tests but the last one to not mutate anything.